### PR TITLE
fixing restart for IBFE with systems for postprocessed variables

### PR DESCRIPTION
**Pull request by cpuelz**

### CODE REVIEW

**Comment by cpuelz** [comment]
when postprocessed variables are brought in from the restart files, it may not be necessary to add the variables again to the system.  in debug mode the assertion for the system "!is_initialized" is tripped, so I added this on the outside of "add_variable".

File: src/IB/IBFEPostProcessor.cpp [hunk 1]
```diff
@@ -104,7 +104,7 @@ IBFEPostProcessor::registerScalarVariable(const std::string& name,
 {
     EquationSystems* equation_systems = d_fe_data_manager->getEquationSystems();
     System& system = equation_systems->add_system<System>(name + " reconstruction system");
-    system.add_variable(name, fe_order, fe_family);
+    if(!system.is_initialized()) system.add_variable(name, fe_order, fe_family);
```
Comment (boyceg) [inline src/IB/IBFEPostProcessor.cpp]: Maybe this should be `if (!is_from_restart)`?

You can find out if a run is from restart via:
```
    RestartManager* restart_manager = RestartManager::getManager();
    bool is_from_restart = restart_manager->isFromRestart();
```

Comment (cpuelz) [reply src/IB/IBFEPostProcessor.cpp]: yes, this is cleaner.

**Comment by boyceg** [comment]
Jenkins retest this please.

**Comment by cpuelz** [comment]
how do I Jenkins retest it?

**Comment by boyceg** [comment]
You make a comment, "Jenkins retest this please." I may need to "white list" you to get it to work. However, all tests are passing, so I will go ahead and merge.

**Comment by boyceg** [comment]
@drwells this probably should also go into 0.3.

**Pull request merged**